.env.test

```
@@ -0,0 +1,20 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+
+TRELLO_API_KEY=FakeAPIKey
+
+TRELLO_API_TOKEN=FakeAPIToken
+
+TRELLO_TODO_LIST_ID=65b23d26b3671d90bde4fb1e
+
+TRELLO_DONE_LIST_ID=65b23d2c7572ec89c6195e9e
+
+TRELLO_BOARD_ID=65b2371c510c35b32bf1f848
```

These variables aren't sensitive, but I'm still a fan of making them blatantly fake like the sensitive ones, for one thing it makes it more likely that we spot what's happened if we accidentally do have real values creeping into our tests, since a long string of digits is easy to spot if you're expecting `fake-todo-id`, e.g.:
```suggestion
TRELLO_TODO_LIST_ID=fake-todo-list

TRELLO_DONE_LIST_ID=fake-done-list

TRELLO_BOARD_ID=fake-board-id
```